packages/arb-bridge-eth/src/utils.ts

```diff
@@ -73,6 +76,55 @@ export const getL2RetryableTicketId = (inboxSequenceNumber: BigInt): Bytes => {
   return res;
 };
 
+export const getL2NitroRetryableTicketId = (
+  event: InboxMessageDeliveredEvent,
+  retryable: RetryableTx,
+  messageSenderAddress: Address
+): Bytes => {
+  const l2ChainId: Bytes = getL2ChainId();
+  const msgNum = padBytes(getBytes(event.params.messageNum), 32);
+  const fromAddress: ByteArray = ByteArray.fromHexString(messageSenderAddress.toHexString());
+  let l1BaseFee: ByteArray = getBytes(BigInt.fromI32(0));
+  if (event.block.baseFeePerGas) {
+    l1BaseFee = getBytes(event.block.baseFeePerGas!);
+  }
+  const l1Value: ByteArray = getBytes(event.transaction.value);
+  const maxFeePerGas: ByteArray = getBytes(retryable.gasPriceBid);
+  const gasLimit: ByteArray = getBytes(retryable.maxGas);
+  const destAddress: ByteArray = ByteArray.fromHexString(retryable.destAddress.toHexString());
```

Comment:
```suggestion
  const destAddressString: string = retryable.destAddress.toHexString();
  const destAddress: ByteArray = ByteArray.fromHexString(destAddressString == "0x0000000000000000000000000000000000000000" ? "0x" : destAddressString);
```

See https://github.com/OffchainLabs/arbitrum-sdk/pull/254